Homework_11/src/testng/BicycleTest.java

```diff
@@ -0,0 +1,54 @@
+package test.java;
+
+import org.testng.annotations.*;
+
+import tat.maven.Bicycle;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by Nick on 30/10/16.
+ */
+public class BicycleTest {
+    Bicycle bicycle;
+    double start;
+
+    @DataProvider(name = "points")
+    public Object[][] getPoints() {
+        return new Object[][]{
+                {2, 0, 2},
+                {3, 0, 3},
+                {Double.NaN, Double.NaN, 9},
```

Comment:
lines 21,22,23 : Values for nagitive tests but used is positive test, will never pass